org-ql-search.el

```diff
@@ -242,6 +242,15 @@ automatically from the query."
   "Insert content for org-ql dynamic block at point according to PARAMS.
 Valid parameters include:
 
+  :scope    The scope to consider for the Org QL query. This can
+            be one of the following:
+
+            `buffer'              the current buffer
+            `org-agenda-files'    all agenda files
+            `org-directory'       all org files
+            `(list-of-files ...)' a list of org files
```

Comment:
I'm glad you listed these choices explicitly.  However, this list-of-files one needs clarification: is it an arbitrary Lisp list?  Arbitrary Lisp code?  A list of strings?  A list of symbols which may evaluate to strings?  etc.

Of course, it *should* probably only accept one string or a list of strings; any other types of value or a list of any other types of value should be rejected.  However, we might consider accepting variables and lists of variables, which would make it easier for the user to refer to sets of files.

Either way, we should be very careful to avoid security issues with executing arbitrary code when the block is updated.  Please see the tests related to the other block arguments for examples.